spec/classes/foreman_proxy_content__pub_dir_spec.rb

```diff
@@ -7,6 +7,16 @@
         facts
       end
 
+      let(:pre_condition) do
```

Comment:
Looking at this and the actual code, I wonder why this is here:
https://github.com/theforeman/puppet-foreman_proxy_content/blob/8f76e215534db857228b56b075598d39afcd0fbe/manifests/pub_dir.pp#L14

If you remove that, the class becomes standalone and this isn't needed.

@timogoebel you added this in c363e7c0b068e7d2d06c5c0698c98f545c1c1f31 but I don't see why.

Reply:
I don't remember. Probably because the class needs some prerequisites the other class sets up to work. Maybe files referenced in the `additional_includes` section?

Reply:
Could we opt to file an issue and come back to it when the Pulp cleanup happens? I'm wanting to be minimally invasive